enterprise/config/buildbuddy.local.yaml

```diff
@@ -13,8 +13,58 @@ storage:
     root_directory: /tmp/buildbuddy_enterprise
   enable_chunked_event_logs: true
 cache:
+  # Sample flow if migration disk -> pebble
+
+  # Begin double writing, and still return data from disk
```

Comment:
Do we also want to be able to configure a double write percentage? For example, if double-write would be too expensive, maybe it makes sense to do it only for digests starting with `ff` or something like that.

Reply:
I think a slow rollout of the apps would be cleaner. If we only double write a %, the dest cache may hold some stale data

Ex. 
T0: 
- Write 5 to src
- <Exceeds double write threshold> Write 5 to dest

T1:
- Write 7 to src
- <Does not exceed double write threshold> No write to dest, dest still holds 5

It may be complicated to detect that dest cache is holding stale data (Ex. if no one reads that key for a while, so the src value never gets copied over or double read)

Reply:
If we wanted a double write % we could implement a stable way to ensure that if one key is double written, it is always double written, but that feels a bit complicated

Reply:
Slow rollout sgtm